en/drafts/originals/creating-guis-in-python-for-digital-humanities-projects.md

```diff
@@ -3,12 +3,12 @@ title: "Creating GUIs in Python for Digital Humanities Projects"
 slug: creating-guis-in-python-for-digital-humanities-projects
 layout: lesson
 collection: lessons
-date: YYYY-MM-DD
+date: 2023-MM-DD
 authors:
 - Christopher Thomas Goodwin
 reviewers:
 - Forename Surname
```

Comment:
@yann-ryan Could you add in the name of the second reviewer? I can't work out their name from the GitHub handle.

Reply:
Done!